UI/Selenium/Helper/Helper.cs

```diff
@@ -24,13 +25,13 @@ public static IWebElement Find(this IWebDriver driver, By by,int timeoutSeconds=
 
                     if (element.Displayed && element.Enabled)
                     {
-                        Logger.Info(" The following element has been found " + element);
+                        Logger.InfoWithDate("The following element has been found " + element);
```

Comment:
What will the log display for element?  It probably needs more refinement to get the element name and the page name.

This is repeated throughout the application and must be fixed everywhere.

Reply:
we now have ScenarioContext passed to ExtensionMethods. This will help to print the test context